path_checkouts_test.go

```diff
@@ -0,0 +1,30 @@
+package openldap
+
+import (
+	"testing"
+
+	"github.com/hashicorp/vault/sdk/logical"
+)
+
+func TestCheckInAuthorized(t *testing.T) {
```

Comment:
Might be a nice candidate for a table driven test, which reduces duplication, and separates data from logic so it's easier to read the test case data.

Reply:
Indeed! I also prefer the table driven tests. These are copied from AD secrets engine, and I left them unmodified to keep the diff small. I can create a task to improve these tests after this PR is merged.